PUP Armor Plates, some recipes w/ Mythril/Mercury

RAIST5150 opened the pull request:
A lot of verify flags out there on these.
Dug up talks at times before 2010, and made some judgement calls.
Notes will be in comments.
Special notes on PUP Armor plates: 
FFXIAH cites 34 AL for first version in discussion, JPWikki agreed on first two versions.
All four tiers of these plates use same alchemy materials/quantities, so I opted to extend 34AL on III/IV as well

<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

RAIST5150 committed:
PUP Armor Plates, some recipes w/ Mythril/Mercury

A lot of verify flags out there on these.
Dug up talks at times before 2010, and made some judgement calls.
Notes will be in comments.
Special notes on PUP Armor plates: 
FFXIAH cites 34 AL for first version in discussion, JPWikki agreed on first two versions.
All four tiers of these plates use same alchemy materials/quantities, so I opted to extend 34AL on III/IV as well

claywar commented:
I'd say leave the comments, but let's leave the recipes disabled for ones that we can't confirm.  AP3/4 have alternate sources to acquire as well, so if it's incorrect its an item that would fall through the cracks if incorrect.

claywar commented:
```
Importing sql/synth_recipes.sql into the database...
ERROR 1136 (21S01) at line 969: Column count doesn't match value count at row 1
Error: Process completed with exit code 1.
```

Xaver-DaRed commented on sql/synth_recipes.sql:
```diff
@@ -954,26 +954,26 @@ INSERT INTO `synth_recipes` VALUES (13023,0,0,0,64,0,0,0,0,0,0,4099,4241,659,666
 INSERT INTO `synth_recipes` VALUES (13037,0,0,0,67,0,0,0,0,0,0,4096,4238,648,662,664,682,850,0,0,0,12427,12439,12439,12439,1,1,1,1,'Bascinet');
 INSERT INTO `synth_recipes` VALUES (13038,0,0,37,67,16,0,0,0,0,0,4096,4238,652,654,719,719,744,0,0,0,18704,18705,18705,18705,1,1,1,1,'Darksteel Hexagun');
--- INSERT INTO `synth_recipes` VALUES (13039,0,0,0,67,255,0,0,0,0,0,4096,4238,654,719,719,744,2764,0,0,0,18737,18737,18737,18737,1,1,1,1,'Fane Hexagun');
+INSERT INTO `synth_recipes` VALUES (13039,0,037,67,16,0,0,0,0,0,4096,4238,654,719,719,744,2764,0,0,0,18737,18737,18737,18737,1,1,1,1,'Fane Hexagun'); -- JPWiki used, others 60+,67SM, 31-45??WW, ??GS
```
Missing a comma here. 037 should probably be 0,37

RAIST5150 replied:
oops... got a little happy with the backspace key.  submitted correction while disabling armor plates III/IV, as JPWikki is not actually citing 34AL on those two yet.

Xaver-DaRed replied:
Could you correct those extra spaces while you are at it? We don't wanna trigger OCD

TeoTwawki replied:
> Could you correct those extra spaces while you are at it? We don't wanna trigger OCD

WHO SUMMONS ME?



:)

RAIST5150 committed:
corrections from review

disabled armor plates III and IV, as no source actually cites 34 ALCH yet.

Correct typo in skill levels for Fane Hexagun.

TeoTwawki commented on sql/synth_recipes.sql:
```diff
@@ -984,50 +984,50 @@ INSERT INTO `synth_recipes` VALUES (13053,0,0,0,70,0,0,0,0,0,0,4096,4238,654,852
 INSERT INTO `synth_recipes` VALUES (13532,0,1993,0,78,0,0,0,0,0,0,4096,4238,2230,2233,18207,0,0,0,0,0,18493,18493,18493,18493,1,1,1,1,'Regiment Kheten');
--- INSERT INTO `synth_recipes` VALUES (13533,0,0,0,78,0,0,0,0,255,0,4096,4238,665,932,2001,2290,4022,0,0,0,8556,8556,8556,8556,12,12,12,12,'Armor Plate IV');
--- INSERT INTO `synth_recipes` VALUES (13534,0,0,255,79,27,0,0,0,0,0,4096,4238,650,652,654,711,0,0,0,0,17250,17261,17261,17261,1,1,1,1,'Matchlock Gun');
+ -- INSERT INTO `synth_recipes` VALUES (13533,0,0,0,78,0,0,0,0,34,0,4096,4238,665,932,2001,2290,4022,0,0,0,8556,8556,8556,8556,12,12,12,12,'Armor Plate IV'); -- JPWiki BUT ??AL--FFXIAH cites 34AL on 1st tier, same mats/qty for II/III/IV versions
```
got a space in front of you dashes

RAIST5150 replied:
gah... never noticed that they fully aligned to left.  got too used to " -- " for comments at end of line.

important thing to remember.

edited out

RAIST5150 committed:
remove blank space at start of disabled lines
